content/en/docs/components/pipelines/operator-guides/configure-object-store.md

```diff
@@ -0,0 +1,404 @@
++++  
+title = "Object Store Configuration"  
+weight = 3  
++++
+
+{{% kfp-v2-keywords %}}
+
+In Kubeflow Pipelines (KFP) are two components that utilize Object store:
+
+* KFP API Server
+* KFP Launcher (aka KFP executor)
+
+The default object store that is shipped as part of the Kubeflow Platform is Minio. However, you can configure a different object store provider with your KFP deployment.
+
+The following diagram provides an simplified overview of how object storage is utilized and configured: 
+
+<img src="/docs/components/pipelines/operator-guides/images/kfp-obj-store-configurations.png"
+alt="KFP Object Store Configuration Overview"
+class="mt-3 mb-3"
+style="width: 75%; height: 75%;">
+
+
+## Prerequisites
+
+* Admin level access to KFP Kubernetes namespace
+* Object Store credentials for a supported provider (see below)
+
+> **Note**: in this doc "**KFP Namespace**" refers to the namespace where KFP is deployed. If KFP is deployed as part of the Kubeflow Platform deployment, this is the `kubeflow` namespace.
+
+## KFP API Server
+
+The KFP API Server uses the object store to store the Pipeline Intermediate Representation (IR).
+
+The list below describe the type of Object Store configurations supported today for API Server. Static credentials here
+refers to long term credentials provided by Object Store providers. 
+
+For [AWS Static Credentials] and other S3 compliant object storage, this consists of an Access Key ID and Secret Access Key ID embedded into the run time environment or, passed as secure parameters to some API. In Google Cloud Storage, this refers to a JSON containing the [GCS APP Credentials].
+
+### API Server Supported providers
+
+| Provider                                     | Supported |
+|----------------------------------------------|-----------|
+| Minio with Static Credentials                | Yes       |
+| AWS S3 with Static Credentials               | Yes       |
+| AWS S3 with IRSA                             | Yes       |
+| S3-Compliant Storage with Static Credentials | Yes       |
+| Google Cloud Storage with Static Credentials | No        |        
+| Google Cloud Storage with App Credentials    | No        |  
+
+### API Server Object Store Configuration
+
+To configure the object store used by the KFP API Server, the configuration depends on whether you are using static credentials, or AWS S3 with IRSA.
```

Comment:
I think IRSA is not the best way to mention. Refer to the full name, then add the acronym in parenthesis.

Worth mentioning that a reference link would be also good.

Reply:
fixed